src/main/java/com/delphix/masking/initializer/maskingApi/BackupDriver.java

```diff
@@ -403,6 +408,39 @@ private void backupUsers() throws ApiCallException {
         }
     }
 
+    private void backupJdbcDrivers() throws ApiCallException {
+        GetSystemInformations getSystemInformations = new GetSystemInformations();
+        apiCallDriver.makeGetCall(getSystemInformations);
+        SystemInformation systemInformation = getSystemInformations.getSystemInformation();
+        String[] versions = systemInformation.getVersion().split("[.]");
+        if(versions.length == 4) {
+            int mejarVersion = 0;
+            int minorVersion = 0;
+            int microVersion = 0;
+            if(StringUtils.isNumeric(versions[0])) {
+                mejarVersion = Integer.parseInt(versions[0]);
+            }
+            if(StringUtils.isNumeric(versions[1])) {
+                minorVersion = Integer.parseInt(versions[1]);
+            }
+            if(StringUtils.isNumeric(versions[2])) {
+                microVersion = Integer.parseInt(versions[2]);
+            }
+            // backup JDBC Drivers only if the release version is 6.0.1.0 or higher
+            if(mejarVersion >= 6 && minorVersion >= 0 && microVersion >= 1) {
```

Comment:
Thanks for doing this!

Can we move this logic to another function so that we can just call it from other places as well?

Maybe in the `run` function, we could gather the version info from the engine and then use it in a check version function.
```
if (versionAbove(6, 0, 1)) {
   // backup jdbc driver
}
```

Reply:
Done.

Reply:
I used this file to generate the system-information classes, but later on, I forgot to remove it. Thanks.